packages/react-app/public/index.html

```diff
@@ -8,12 +8,22 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
 
     <meta property="og:title" content="Speed Run Ethereum" />
-    <meta name="description" content="Learn how to build on Ethereum; the superpowers and the gotchas.">
+    <meta name="description" content="Learn how to build on Ethereum; the superpowers and the gotchas." />
     <meta property="og:description" content="Learn how to build on Ethereum; the superpowers and the gotchas." />
-    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:card" content="summary_large_image" />
     <meta property="og:image" content="%PUBLIC_URL%/thumbnail.png" />
     <meta property="twitter:image" content="%PUBLIC_URL%/thumbnail.png" />
 
+    <meta property="fc:frame" content="vNext" />
+    <meta property="fc:frame:image" content="%PUBLIC_URL%/thumbnail.png" />
+
+    <meta property="fc:frame:button:1" content="🛠️ GitHub" />
+    <meta property="fc:frame:button:1:action" content="link" />
+    <meta property="fc:frame:button:1:target" content="https://github.com/scaffold-eth/se-2-challenges" />
+
+    <meta property="fc:frame:button:2" content="⚡️ Start" />
+    <meta property="fc:frame:post_url" content="https://buidlguidl-v3.ew.r.appspot.com/api/frame?id=0" />
```

Comment:
If you want to test the frame locally, you'll need to change this to the local backend url.